lab/dp/problems/sum_query/main.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright 2020 Nichita Radu <radunichita99@gmail.com>
+
+// SumQuery - Programare dinamica
+// Complexitate O(M * N)
+
+// https://leetcode.com/problems/range-sum-query-2d-immutable/
+
+#include <bits/stdc++.h>
+
+class Task {
+    // N si M sunt dimensiunile matricei, K e numarul de query-uri
+    int N, M, K;
+
+    // matricea de elemente
+    std::vector<std::vector<int>> matrix;
+
+    // query-urile de forma <(start_x, start,y), (finish_x), (finish_y)>.
```

Comment:
```suggestion
    // query-urile de forma <(start_x, start,y), (finish_x, (finish_y)>.
```

These 4 values also need to be renamed.